components/rpc/invoker/mosn/channel/connpool.go

```diff
@@ -59,8 +62,8 @@ func (w *wrapConn) close() error {
 	return err
 }
 
-// im-memory fake conn pool
-func newConnPool(
+// NewConnPool im-memory fake conn pool
+func NewConnPool(
```

Comment:
why expose this function?

Reply:
i add more complete explanation

Reply:
why change method name

Reply:
It's amazing. In fact, I didn't change this. I may have pressed the shortcut key. I've changed it back